pkg/koordlet/metricsadvisor/collector.go

```diff
@@ -57,8 +57,9 @@ type Collector interface {
 }
 
 type contextRecord struct {
-	cpuTick uint64
-	ts      time.Time
+	cpuTick  uint64
```

Comment:
is `cpuTick` still needed?

Reply:
@saintube can you double-check this one?

Reply:
lgtm